### ISSUE TITLE
Get/Print "logits" values of generated tokens in mlc-llm

### ISSUE THREAD

**Issue by chanijjani** [ballinyouup/mlc-cli#1]
- It will be best if a Python API is provided for Linux environments.
